install_config/installing-pipelines.adoc

```diff
@@ -18,6 +18,11 @@ This guide walks cluster administrators through the process of installing the {p
 * You have installed xref:../tkn_cli/installing-tkn.adoc#installing-tkn[{pipelines-shortname} (`tkn`) CLI] on your local system.
 * Your cluster has the link:https://docs.openshift.com/container-platform/latest/installing/cluster-capabilities.html#marketplace-operator_cluster-capabilities[Marketplace capability] enabled or the Red Hat Operator catalog source configured manually.
 
+[NOTE]
+====
+In a cluster with both Windows and Linux nodes, the {pipelines-title} can only run on Linux nodes.
```

Comment:
```suggestion
In a cluster with both Windows and Linux nodes, the {pipelines-title} can run on only Linux nodes.
```
- Is the note missing the word "Operator" after {pipelines-title}? If not, I'd remove the "the" from before {pipelines-title}.
- Per IBM Style, place "only" as close to the word it modifies as possible: https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only